core/templates/dev/head/pages/story-viewer-page/chapters-list/story-viewer-chapters-list.directive.ts

```diff
@@ -85,26 +87,53 @@ angular.module('oppia').directive('storyViewerChaptersList', [
 
           $anchorScroll.yOffset = -80;
 
-          ctrl.setIconHighlight = function(index) {
-            ctrl.activeHighlightedIconIndex = index;
+          ctrl.isVisibleCard = function(storyNode) {
```

Comment:
Should probably phrase it as isCardVisible (it reads better).

That said, no idea what "card" refers to here -- typically "card" means "state". Could you clarify the name please?

Reply:
I meant the exploration summary tile, whether it is visible when hovered on or clicked. Will rename it `isSummaryVisible`.

Reply:
isSummaryTileVisible

Reply:
Done